### ISSUE TITLE
Dark mode, light mode media

### ISSUE THREAD

**Issue by GrosSacASac** [validator/validator#1142]
URL being validated or code to reproduce error:
```
<link href="../css/tomorrow.css" rel="stylesheet" media="(prefers-color-scheme: light)">
```

```
Error: Bad value (prefers-color-scheme: light) for attribute media on element link: Expected a CSS media feature but saw prefers-color-scheme instead.
```

**Comment by sideshowbarker** [comment]
I can’t actually reproduce this.

I tested with the following minimal case:

```html
<!DOCTYPE html>
<html lang="">
<title>Test</title>
<link href="../css/tomorrow.css" rel="stylesheet" media="(prefers-color-scheme: light)">
```

…but the checker reports no errors for that.

**Comment by GrosSacASac** [comment]
Ok I just found out there are 2 validators
https://validator.w3.org/nu/
and 
https://validator.nu/

**Comment by GrosSacASac** [comment]
I have another warning where I put a link in the body
```
Error: A link element must not appear as a descendant of a body element unless the link element has an itemprop attribute or has a rel attribute whose value contains dns-prefetch, modulepreload, pingback, preconnect, prefetch, preload, prerender, or stylesheet.

From line 147, column 1; to line 147, column 80

omments">↩<link href="https://letz.social/blog-comments-sac/webmention" rel="webmention">↩<nosc
```
Is the warning correct ?

**Comment by GrosSacASac** [comment]
https://www.w3.org/TR/webmention/#sender-discovers-receiver-webmention-endpoint

I found nothing about the fact that the link must be in the head

**Comment by sideshowbarker** [comment]
> I have another warning where I put a link in the body
> 
> ```
> Error: A link element must not appear as a descendant of a body element unless the link element has an itemprop attribute or has a rel attribute whose value contains dns-prefetch, modulepreload, pingback, preconnect, prefetch, preload, prerender, or stylesheet.
> 
> From line 147, column 1; to line 147, column 80
> 
> omments">↩<link href="https://letz.social/blog-comments-sac/webmention" rel="webmention">↩<nosc
> ```
> 
> Is the warning correct ?
>
> [w3.org/TR/webmention/#sender-discovers-receiver-webmention-endpoint](https://www.w3.org/TR/webmention/#sender-discovers-receiver-webmention-endpoint)
> 
> I found nothing about the fact that the link must be in the head

Please raise a separate issue for that.

**Comment by GrosSacASac** [comment]
but I am not sure if it is an issue
